Makefile

```diff
@@ -174,6 +153,10 @@ run-healthy:
 run-status:
 	@USHIFT_IMAGE=${USHIFT_IMAGE} ISOLATED_NETWORK=${ISOLATED_NETWORK} LVM_DISK=${LVM_DISK} LVM_VOLSIZE=${LVM_VOLSIZE} VG_NAME=${VG_NAME} ./src/cluster_manager.sh status
 
+.PHONY: env
```

Comment:
We should make this target dependent on `run-ready`, or we might get errors like the following, while the cluster is initializing.

```
$ make env
Copying kubeconfig from microshift-okd-1...
Error: "/var/lib/microshift/resources/kubeadmin/lenovo.giganet.local/kubeconfig" could not be found on container microshift-okd-1: no such file or directory
make: *** [Makefile:182: env] Error 125

```